core/controllers/reader.py

```diff
@@ -108,7 +110,7 @@ def classify_string_classifier_rule(state, normalized_answer):
         labels = sc.predict([normalized_answer])
         predicted_label = labels[0]
         if (predicted_label !=
-                lda_string_classifier.LDAStringClassifier.DEFAULT_LABEL):
+                sc.DEFAULT_LABEL):
```

Comment:
Move to previous line and remove the parentheses.

Also, should this be a global concept, and not unique to the string classifier? If so, move DEFAULT_LABEL to feconf and have the string classifier refer to that.